src/workos.spec.ts

```diff
@@ -249,14 +296,35 @@ describe('WorkOS', () => {
     });
 
     describe('when the entity is null', () => {
-      it('sends a null body', async () => {
+      it('sends an empty string body', async () => {
```

Comment:
In fetch environments we can't have the body be null due to how Deno works:

https://github.com/workos/workos-node/pull/1038/files#diff-eb8ece7bf63e57536fd07d4e0ef7369670a03714e326e2a55c1c4a947993fb5dR119-R121